src/ebay-page-notice/page-notice.tsx

```diff
@@ -1,44 +1,70 @@
-import React, { FC, ReactElement } from 'react'
+import React, { Dispatch, FC, ReactElement, useState } from 'react'
 import NoticeContent from '../common/notice-utils/notice-content'
 import { EbayNoticeContent } from '../ebay-notice-base/components/ebay-notice-content'
 import { EbayIcon, Icon } from '../ebay-icon'
 
 type Props = React.HTMLProps<HTMLElement> & {
     status?: 'general' | 'attention' | 'confirmation' | 'information',
-    'aria-label'?: string;
+    'aria-label'?: string,
+    a11yDismissText: string,
+    onDismissed?: Dispatch<boolean>
 };
 
 const EbayPageNotice: FC<Props> = ({
     status = 'general',
     children,
+    onDismissed,
+    a11yDismissText,
     'aria-label': ariaLabel,
     ...rest
 }) => {
+    const [dismissed, setDismissed] = useState(false)
     const childrenArray = React.Children.toArray(children)
     const content = childrenArray.find((child: ReactElement) => child.type === EbayNoticeContent) as ReactElement
 
     if (!content) {
         throw new Error(`EbayPageNotice: Please use a EbayNoticeContent that defines the content of the notice`)
     }
 
+    const handleDismissedClicked = () => {
+        setDismissed(true)
+        if (onDismissed) {
+            onDismissed(true)
+        }
+    }
+
     return (
-        <section
-            {...rest}
-            aria-labelledby={`${status}-status`}
-            className={`page-notice ${status !== `general` ? `page-notice--${status}` : ``}`}
-            role="region">
-            {status !== `general` ? (
-                <div className="page-notice__header" id={`${status}-status`}>
-                    <EbayIcon
-                        name={`${status}FilledSmall` as Icon}
-                        a11yText={ariaLabel}
-                        a11yVariant="label"
-                    />
-                </div>
-            ) : null}
-            <NoticeContent {...content.props} type="page" />
-            {children}
-        </section>
+        <>
```

Comment:
We don't need to use a fragment for this use case:

```tsx
return !dismissed && (
   <section></section>
)
```

I usually prefer to have a separate if and return null, to make it more explicity, but that is personal opinion, feel free to follow or not:

```tsx
if (dismissed) {
  return null
}

return (
   <section></section>
)
```

Reply:
I usually do
```
return !dismissed ? null : <section>...</section>
```

My point is: don't return `undefined`, components have to return either a `ReactElement` or `null`.

Reply:
Good feedback. Updated.